packages/components-native/src/Divider/DividerHorizontal.style.ts

```diff
@@ -1,17 +1,22 @@
 import { StyleSheet } from "react-native";
 import { tokens } from "../utils/design";
 
-export const styles = StyleSheet.create({
+export const horizontalStyles = StyleSheet.create({
   base: {
-    height: tokens["space-minuscule"],
+    height: "1px",
     margin: 0,
-    borderBottomWidth: tokens["border-base"],
+    width: "auto",
     borderBottomColor: tokens["color-border"],
+    borderBottomWidth: tokens["border-base"],
   },
   large: {
     borderBottomWidth: tokens["border-thick"],
     opacity: 0.875,
   },
+  larger: {
+    borderBottomWidth: tokens["border-thicker"],
+    opacity: 0.625,
+  },
   largest: {
     borderBottomWidth: tokens["space-small"],
```

Comment:
Would also suggest using `border-thickest` ([source](https://atlantis.getjobber.com/?path=/docs/design-borders--docs#border-widths)) here!

Reply:
wait.. shouldn't the `largest` border be the thickest?

Reply:
Yes, that was my suggestion (L21?)

Reply:
yes you are right sorry!! I misread and I was so confused 😂